bundles/CoreBundle/src/Command/Document/CleanupCommand.php

```diff
@@ -33,10 +33,9 @@ class CleanupCommand extends AbstractCommand
         'folder',
         'hardlink',
         'email',
-        'newsletter',
     ];
 
-    private const PROTECTED_DOCUMENT_TYPES = ['page', 'link', 'snippet', 'folder', 'hardlink', 'email', 'newsletter'];
+    private const PROTECTED_DOCUMENT_TYPES = ['page', 'link', 'snippet', 'folder', 'hardlink', 'email'];
```

Comment:
Also add step to remove newsletter types from document tables [here](https://github.com/pimcore/pimcore/blob/4f37f72d95e41a2c4720f4da340aa0f091a1a91a/doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md#web2print-1)

Reply:
Yes we should absolutely do this here too if there are not planning to use the newsletter.